spring-cloud-consul-core/src/main/java/org/springframework/cloud/consul/ConsulAutoConfiguration.java

```diff
@@ -82,9 +82,13 @@ public ConsulEndpoint consulEndpoint(ConsulClient consulClient) {
 
 		@Bean
 		@ConditionalOnMissingBean
-		@ConditionalOnEnabledHealthIndicator("consul")
-		public ConsulHealthIndicator consulHealthIndicator(ConsulClient consulClient) {
-			return new ConsulHealthIndicator(consulClient);
+		@ConditionalOnProperty(
+				value = { "spring.cloud.consul.health-indicator.enabled",
+						"management.health.consul.enabled" },
```

Comment:
I followed suit to add the enabled property on the new config props as the DiscoveryClientHealthIndicator is done that way. However, I also respect the typical actuator health enabled property of "`management.health.ID.enabled`". 

Not sure if we want to 
1. keep both
2. only use the Actuator "management.." one
3. use the new one and find a way to deprecate usage of the other (no way other than docs I think though).

Reply:
This conditional shouldn't change. It should be `@ConditionalOnEnabledHealthIndicator("consul")`.